_layouts/default.html

```diff
@@ -1,3 +1,6 @@
+---
+large_logo: false
```

Comment:
This is needed so that this PR doesn't change the Homepage. We can remove this when implementing #65.